scanner/src/test/kotlin/scanners/scancode/ScanCodeResultParserTest.kt

```diff
@@ -211,6 +216,48 @@ class ScanCodeResultParserTest : FreeSpec({
                 )
             }
         }
+
+        "for output format 3.0.0 should" - {
```

Comment:
Consider below alternative which (1) tests both boundary cases, uses (2) `MAX_SUPPORTED_OUTPUT_FORMAT_VERSION` so the test needs not be updated when 
the supported output version is changed and (3) weakens the assertion so that changes to the error message do not require updating the test.

```
"generateSummary()" should {
     "not create a warning for highest supported output version" {
     	val summary = createOutputJson(SpdxConstants.NONE,MAX_SUPPORTED_OUTPUT_FORMAT_VERSION))
        summary.issues.count { it.severity == Severity.WARNING } shouldBe 0
     }
     
     "create a warning for lowest unsupported output version" {
        val version = Semver(MAX_SUPPORTED_OUTPUT_FORMAT_VERSION).withIncPatch()
     	val summary = createOutputJson(SpdxConstants.NONE, version.toString())
     
        summary.issues.count { it.severity == Severity.WARNING } shouldBe 1
     }
}

fun createOutputJson(outputFormatVersion: String): JsonNode {
    ...
}
```

Reply:
> "not create a warning for highest supported output version"

I believe that case is sufficiently covered by other tests.

> "create a warning for lowest unsupported output version"

This actually pointed me at the fact that we should consider less-than-major changes in the output format version compatible. I'll change the implementation accordingly.